std/math/operations.d

```diff
@@ -1006,7 +1006,13 @@ if (isFloatingPoint!(X))
 
    Warning:
         This template is considered out-dated. It will be removed from
-        Phobos in 2.106.0. Please use $(LREF isClose) instead.
+        Phobos in 2.106.0. Please use $(LREF isClose) instead. To (almost)
+        keep the current behaviour of `approxEqual(a, b)` use
```

Comment:
Reword the phrase to: "To achieve a similar behavior to `aproxEqual(a, b)` use .."